### PR TITLE
Add missing Gradle dependencies in Application Developer Guide

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/eventQueue.rst [hunk 1]
```diff
@@ -396,9 +396,19 @@ Use
 The `Event Queue API Module`_ must be added to the :ref:`module.ivy <mmm_module_description>` of the MicroEJ
```
Comment (bguedas) [inline ApplicationDeveloperGuide/eventQueue.rst:396]: This line must be updated too. One should talk about the project build file, or something else generic

File: ApplicationDeveloperGuide/runtime.rst [hunk 2]
```diff
@@ -65,9 +65,20 @@ The `EDC API Module`_ must
 be added to the :ref:`module.ivy <mmm_module_description>` of the Application
```
Comment (bguedas) [inline ApplicationDeveloperGuide/runtime.rst:65]: References module.ivy

File: ApplicationDeveloperGuide/runtime.rst [hunk 3]
```diff
@@ -108,9 +119,20 @@ allows:
 Add the following dependency to the :ref:`module.ivy <mmm_module_description>` of the Application
```
Comment (bguedas) [inline ApplicationDeveloperGuide/runtime.rst:119]: Same as above

**Review by bguedas — CHANGES_REQUESTED** [review]

File: ApplicationDeveloperGuide/eventQueue.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/eventQueue.rst]: Done.

File: ApplicationDeveloperGuide/runtime.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/runtime.rst]: Done.

File: ApplicationDeveloperGuide/runtime.rst (hunk 3 above)
Comment (tdelhomenie-microej) [reply ApplicationDeveloperGuide/runtime.rst]: Done.

**Review by bguedas — APPROVED** [review]

**Pull request merged**